### PR TITLE
Custom cell documentation added

### PR DESCRIPTION
**Pull request by GrantBrown1994**
Hyperlinks may need to be updated after merged. Currently they point to my fork.

### CODE REVIEW

File: DOC/source/datasheet/sofa_chd/custom_cells/custom_cells.rst [hunk 1]
```diff
@@ -0,0 +1,133 @@
+=========================================================
```
Comment (tangxifan) [inline DOC/source/datasheet/sofa_chd/custom_cells/custom_cells.rst:1]: Can you register the rst file to the index file? As such, the rst will be included in the doctree. We can check the compiled version in CI.

**Review by tangxifan — CHANGES_REQUESTED** [review]

File: DOC/source/datasheet/sofa_chd/custom_cells/custom_cells.rst (hunk 1 above)
Comment (GrantBrown1994) [reply DOC/source/datasheet/sofa_chd/custom_cells/custom_cells.rst:1]: okay ill look into this

**Comment by GrantBrown1994** [comment]
Updated rst tree, ran make html and make livehtml, manually checked and looked fine. I will update if a problem with CI occurs and a request is made.

**Comment by ganeshgore** [comment]
There is a conflicting file, merge with master and resolve.

**Comment by GrantBrown1994** [comment]
Updated

**Comment by tangxifan** [comment]
Look good for now. Thanks @GrantBrown1994

**Review by tangxifan — APPROVED** [review]

**Pull request merged**